Formula/libdill.rb

```diff
@@ -0,0 +1,54 @@
+class Libdill < Formula
+  desc "Structured concurrency in C"
+  homepage "http://libdill.org"
+  url "https://github.com/sustrik/libdill/archive/1.6.tar.gz"
+  sha256 "2539d3440971c876b117bc87295b0caabc8c8ec5f544856ca9e74ed12705ea51"
+
```

Comment:
maybe add a link to `head` with `head "https://github.com/sustrik/libdill.git"`?

Reply:
let's not